content/release-10-15-0/devices-10-15-0-bundle/10-15-0-377.md

```diff
@@ -0,0 +1,37 @@
+---
+weight: 19
+title: Release 10.15.0.377
+layout: bundle
+---
+
+<!--10.15.0.369 - 10.15.0.377-->
+
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>LWM2M</td>
+<td>Custom actions were not always executed if a LWM2M device used a SEND operation to report the whole LWM2M object data. This issue is now resolved and the custom actions are always executed for the resource instances for which they are defined.</td>
```

Comment:
```suggestion
<td>Custom actions were not always executed if an LWM2M device used a SEND operation to report the whole LWM2M object data. This issue is now resolved and the custom actions are always executed for the resource instances for which they are defined.</td>
```

Reply:
i find two occurrences
i guess it's up for debate ("a lightweight ..." vs " an EL DOUBLE-U...")